db/migrate/20220422223440_add_api_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddApiTokenToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :api_token, :string
+  end
+end
```

Comment:
From the migration `20220414094719_create_likes.rb` file, please delete one end that makes the migration fail.